### ISSUE TITLE
Copy from My Screens Shows Invalid Date for all screens

### ISSUE THREAD

**Issue by DrTedGit** [thunkable/thunkable-issues#770]
### Platform
Thunkable X Block Screen

### Issue/Request Origin
https://community.thunkable.com/t/my-screens-invalid-date/1031826?u=drted

### Steps to Reproduce
Adding a screen (all created before I had a PRO account)

### Project Link
https://x.thunkable.com/copy/c2cbd5b7b81255aaf80957a1f94032b0

### Expected Behavior
Some sort of date
### Actual Behavior
![image](https://user-images.githubusercontent.com/66176281/101763498-b965eb80-3a93-11eb-9ada-9632539a2f95.png)

**Comment by tatiang** [comment]
Yep, this just started happening for me. I've been Pro for a while. When I publish a project as a web app it now tells me "last published on invalid date."
